mopidy/audio/utils.py

```diff
@@ -98,5 +98,5 @@ def disconnect(self, element, event):
 
     def clear(self):
         """Clear all registered signal handlers."""
-        for element, event in self._ids.keys():
+        for element, event in list(self._ids):
```

Comment:
The `for` loop will make sure the iterator is used, so no need for explicit `list()` here I believe.

Reply:
It's required since we are modifying the underlying iterable and will error in py3 otherwise.

Reply:
Aha, that makes sense.